model/src/channel/message/kind.rs

```diff
@@ -29,6 +29,7 @@ pub enum MessageType {
     ThreadStarterMessage = 21,
     GuildInviteReminder = 22,
     ContextMenuCommand = 23,
+    AutoModerationAction = 24,
```

Comment:
Could this be documented?

Reply:
What do you mean by documented, [documented by discord](https://discord.com/developers/docs/resources/channel#message-object-message-types) or there is missing some docs.rs documentation?

Reply:
See https://github.com/twilight-rs/twilight/pull/1792/files#diff-3cfd10dabdf3a2abdea13721cf403393900683604e5fa1a12945c7006e1a1d52